Templates/DefaultProject/Template/ExportScripts/export_standalone_monolithic_engine_centric.py

```diff
@@ -0,0 +1,161 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+
+import argparse
+import glob
+import logging
+import os
+import o3de.export_project as exp
+import pathlib
+import sys
+import time
+
+from o3de.export_project import process_command
+from o3de import manifest
+
+project_json_data = manifest.get_project_json_data(project_path=o3de_context.project_path)
+project_name = project_json_data.get('project_name')
+host_asset_platform = exp.determine_host_asset_platform()
+parser = argparse.ArgumentParser(prog='Exporter for standalone builds',
+                                 description = "Exports a project as standalone to the desired output directory with release layout. "
+                                                "In order to use this script, the engine and project must be setup and registered beforehand. ")
+parser.add_argument('-out', '--output-path', type=pathlib.Path, required=True, help='Path that describes the final resulting Release Directory path location.')
+parser.add_argument('-cfg', '--config', type=str, default='profile', choices=['release', 'profile'],
+                        help='The CMake build configuration to use when building project binaries. If tool binaries are built with this script, they will use profile mode.')
+parser.add_argument('-aof', '--archive-output-format',
+                        type=str,
+                        help="Format of archive to create from the output directory",
+                        choices=["zip", "gzip", "bz2", "xz"], default="zip")
+parser.add_argument('-sl', '--seedlist', type=pathlib.Path, dest='seedlist_paths', action='append',
+                        help='Path to a seed list file for asset bundling. Specify multiple times for each seed list.')
+parser.add_argument('-gf', '--gem-folder-name-to-copy', type=str, dest='gem_folder_names_to_copy', action='append',
+                        help="The name of a Gem folder to include in release. Only specify folder name. This should match what is in the project directory.")
```

Comment:
Can't we extrapolate the gem folder name for the project from the project settings?

Reply:
We could, but the question is "which one?". From what I saw with MPS, that explicitly includes only the AWSCore folder. This is meant to allow users to manually specify what folders they know they want to include (incase existing cmake build processes do not cover it)

Reply:
Maybe allow this parameter to be a file pattern as well (like `MPS*`)

Reply:
A pattern may be a good compromise, but there's currently an issue with this:

Right now the gem folder name is used to specify the glob pattern of files to copy (which your suggestion would match), but also the folder destination in the output release directory for gems.

That may require splitting this up into "gem-patterns-to-copy" and "gem-folder-output" variables, but  that would make this clunkier for sure.

This is what the current code for copying files from gem folders looks like for context:
```

for gem_folder_name in args.gem_folder_names_to_copy:
    output_gem_path = args.output_path / 'Gems'/ gem_folder_name
    os.makedirs(output_gem_path, exist_ok=True)
    for file in glob.glob(str(pathlib.PurePath(mono_build_path / 'bin' / args.config / 'Gems' / gem_folder_name / '*.*'))):
        shutil.copy(file, output_gem_path)
```

Reply:
Instead of using the gem folder name to copy an more exact approach is to use the registered gem name.
The o3de CLI [manifest.get_registered](https://github.com/o3de/o3de/blob/development/scripts/o3de/o3de/manifest.py#L1132-L1139) function can use the gem name to determine the path to gem root folder.
Once that is found the [pathlib.PurePath.name](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.name) could be used to extract the current the folder name